site/assets/scss/_custom.scss

```diff
@@ -2,6 +2,32 @@
 //
 // SPDX-License-Identifier: MIT
 
+/* Code blocks */
+
+div.code-toolbar .toolbar {
```

Comment:
Why do we need these styles? Can we use default?

Reply:
Styles are needed to correct the location and transparency of the `copy` button, make the background color of code blocks the same as without specifying the language, fixes the display of code blocks having a `console` language in the desktop and mobile versions

no styles:
![image](https://user-images.githubusercontent.com/54434686/146518212-7b462843-de7c-4e90-86fe-927eb6118757.png)

with styles:
![image](https://user-images.githubusercontent.com/54434686/146518267-69530640-a88f-4a0c-ae43-fcfdce13855f.png)